src/App.vue

```diff
@@ -1,62 +1,128 @@
 <template>
-    <div id="app" class="h-100 mb-3">
-        <nav class="navbar  navbar-expand-lg  navbar navbar-dark bg-dark">
-            <ul class="navbar-nav mr-auto">
-                <li class="nav-item active">
-                    <a class="nav-link" @click="mode = 'editor'" href="#">{{$t('Editor')}}</a>
-                </li>
-                <li class="nav-item">
-                    <a class="nav-link" @click="mode = 'preview'" href="#">{{$t('Preview')}}</a>
-                </li>
-                <li class="nav-item">
-                    <a class="nav-link" @click="openComputedProperties" href="#">{{$t('Computed Properties')}}</a>
-                </li>
-                <li class="nav-item">
-                    <a class="nav-link" @click="openCustomCSS" href="#">{{$t('Custom CSS')}}</a>
-                </li>
-            </ul>
-        </nav>
+    <div id="app" class="card h-100">
+      <div class="card-header">
+        <div class="row">
+          <div class="col">
+            <div class="btn-group btn-group-sm" role="group" aria-label="Basic example">
```

Comment:
This whole `btn-group` block can be replaced with:

```html
<b-button-group size="sm">
  <b-button :variant="displayBuilder? 'outline-secondary' : 'secondary'" @click="mode = 'editor'">{{ $t('Design') }}</b-button>
  <b-button :variant="!displayBuilder? 'outline-secondary' : 'secondary'" @click="mode = 'preview'">{{ $t('Preview') }}</b-button>
</b-button-group>
```

This uses the available `bootstrap-vue` components instead of classes. Updating the style based on `variant` means the `isBuilderActive` and `isPreviewActive` computed properties can be removed; these computed properties were named in such a way that it seemed like they should return booleans.

The current `aria-label` provides no additional value.

We can use translation! 🎉

Since this is a toggle, removing `isPreviewActive` and using `isBuilderActive` and `!isBuilderActive` may simply be enough.